2.03 description in doc link 357

amy-silcock opened the pull request:
Adding Extra Documentation as per: https://github.com/IATI/IATI-Schemas/issues/357

amy-silcock committed:
Add description to do link in activities

amy-silcock committed:
2.03 add description in doc act and org

amy-silcock committed:
Merge branch 'Add_doc_links_in_results_359' into 2.03-description-in-doc-link-357

amy-silcock committed:
Adding in missing comment element

PetyaKangalova committed:
Changes to example for description/ narrative

I have added a bit more information in the narrative section to make it clearer

PetyaKangalova committed:
Small typo

amy-silcock reviewed (commented):
@PetyaKangalova Looks good

dalepotter commented:
Changed base branch in order to force a refresh to the commit and file diff. This PR previously included changes already in the base branch.

dalepotter commented on .gitignore:
```diff
@@ -4,3 +4,4 @@ en/_build
 */html
 
 
+en/activity-standard/activity-standard-example-annotated.xml
```
I'm not quite sure why this file has been added to `.gitignore`...?

dalepotter committed:
Merge branch 'version-2.03dev' into 2.03-description-in-doc-link-357

dalepotter commented on en/organisation-standard/iati-organisations/iati-organisation/document-link.rst:
```diff
@@ -28,3 +28,6 @@ Changelog
 ^^^^
 Removed ``@language`` attribute from, and introduced an new ``language`` child element to, the ``document-link`` element.
 
+2.03
```
I think the convention is to list changes reverse-chronologically - so here the `2.03` change would be listed before the `1.02` change.

dalepotter committed:
Remove example IATI xml file from .gitignore

It appears that this file was added to .gitignore in error during the v2.03 documentation process.

dalepotter committed:
Swap order of changelog entries

Changelog entries should be in reverse chronological order.

dalepotter committed:
Fix name of element that was added: narrative -> description
